_digest/october-2020/5-jobs-in-medical-field.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: 5 Jobs in the Medical Field You Probably Haven’t Thought About
+date: 10/04/2020
+releaseBatch: October 2020
+summary: “When I grow up, I want to be a gerontologist,” said no one, ever. I highly doubt that even people who are gerontologists right now knew that such a career even existed as a kid. Who can blame them though? No one talks about them. Let’s take a closer look at 5 different jobs in the medical field you probably haven’t thought about.
+authors: ["Keshini Ramchurreetoo"]
+tags: ["Professions"]
+---
+<div>
+
+
+<p> “When I grow up, I want to be a gerontologist,” said no one, ever. I highly doubt that even people who are gerontologists right now knew that such a career even existed as a kid. Who can blame them though? No one talks about them. Let’s take a closer look at 5 different jobs in the medical field you probably haven’t thought about. </p>
+
+<p> <strong>#1; Cytotechnologist</strong> </p>
+
+<p> What did the cell say to his sibling when she stepped on his shoe? Ouch, mitosis! Cytotechnologists prepare and evaluate tissue and human cell samples through a microscope to see if they carry different diseases including (but not limited to); cancer, bacterial infections, and leukemia. To become a cytotechnologist, you will require a minimum of a bachelor’s degree from an accredited cytotechnology program. The average salary for a cytotechnologist in Canada is $31.79 per hour. If you are independent, interested in cytology, microscopic work, and are comfortable with a high degree of responsibility, being a cytotechnologist is something you should look into. </p>
```

Comment:
I think you should be able to use an ordered list with p tags alongside the list items
```
<ol> 
  <li>Cytotechnologist</li> 
  <p> ... </p> 
  <li>Perfusionist</li> 
  <p> ... </p> 
</ol> 
```
which will look like 
```
1. Cytotechnologist
    (paragraph about this) 
2. Perfusionist 
    (paragraph about this)
```